tests/phpunit/unit/api/options/includes/class-options-test-case.php

```diff
@@ -66,28 +68,30 @@ protected function tearDown() {
 	 * Setup dependency function mocks.
 	 */
 	protected function setup_function_mocks() {
-		Monkey\Functions\when( 'add_meta_box' )->alias( function( $id, $title, $callback, $screen = null, $context = 'advanced', $priority = 'default', $callback_args = null ) {
-			global $wp_meta_boxes;
+		Monkey\Functions\when( 'add_meta_box' )->alias(
+			function( $id, $title, $callback, $screen = null, $context = 'advanced', $priority = 'default', $callback_args = null ) {
+					global $wp_meta_boxes;
 
-			if ( empty( $screen ) ) {
-				$screen = 'beans';
-			}
+				if ( empty( $screen ) ) {
+					$screen = 'beans';
+				}
 
-			// phpcs:ignore WordPress.WP.GlobalVariablesOverride.OverrideProhibited -- Mocking global here for tests.
-			$wp_meta_boxes = [
-				$screen => [
-					$context => [
-						$priority => [
-							$id => [
-								'id'       => $id,
-								'title'    => $title,
-								'callback' => $callback,
-								'args'     => $callback_args,
+			   // phpcs:ignore WordPress.WP.GlobalVariablesOverride.OverrideProhibited -- Mocking global here for tests.
+					$wp_meta_boxes = [
```

Comment:
Can you please check that the `if` statement, the comment and this line all start at the same tab position?